docs/openapi3.yaml

```diff
@@ -2034,3 +2040,59 @@ components:
             timeout:
               description: timout to smtp server in milliseconds
               type: number
+        benchmark_config:
+          additionalProperties: false
+          type: object
+          description: Simple mail transfer protocol.
+          required:
+          - threshold
+          - threshold_webhook_url
+          - benchmark_weights
+          properties:
+            threshold:
+              type: number
+              description: benchmark threshold
```

Comment:
please write a description that below this threshold test is considered as failed and webhook will be sent to the threshold_webhook_url